bundle/tests/scorecard/kuttl/runtime_pullpolicy/00-assert.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: example-runtime-component
```

Comment:
It'll be good to add a check for Deployments as well - that it has the right imagePullPolicy